src/node/rpc/forwarder.h

```diff
@@ -77,7 +77,8 @@ namespace ccf
 
       ForwardedHeader msg = {ForwardedMsg::forwarded_cmd, self};
 
-      return n2n_channels->send_encrypted(to, plain, msg);
+      return n2n_channels->send_encrypted(
+        NodeMsgType::forwarded_msg, to, plain, msg);
```

Comment:
This tag should be coming from the msg header to reduce the possibility of a bug in the future